core/controller/src/main/scala/whisk/core/controller/ApiUtils.scala

```diff
@@ -201,30 +204,33 @@ trait ReadOps extends Directives with Logging {
             ma: Manifest[A]) = {
         onComplete(factory.get(datastore, docid)) {
             case Success(entity) =>
-                info(this, s"[PROJECT] entity success")
+                logging.info(this, s"[PROJECT] entity success")
                 complete(OK, project(entity))
             case Failure(t: NoDocumentException) =>
-                info(this, s"[PROJECT] entity does not exist")
+                logging.info(this, s"[PROJECT] entity does not exist")
                 terminate(NotFound)
             case Failure(t: DocumentTypeMismatchException) =>
-                info(this, s"[PROJECT] entity conformance check failed: ${t.getMessage}")
+                logging.info(this, s"[PROJECT] entity conformance check failed: ${t.getMessage}")
                 terminate(Conflict, conformanceMessage)
             case Failure(t: ArtifactStoreException) =>
-                info(this, s"[PROJECT] entity unreadable")
+                logging.info(this, s"[PROJECT] entity unreadable")
                 terminate(InternalServerError, t.getMessage)
             case Failure(t: Throwable) =>
-                error(this, s"[PROJECT] entity failed: ${t.getMessage}")
+                logging.error(this, s"[PROJECT] entity failed: ${t.getMessage}")
                 terminate(InternalServerError)
         }
     }
 }
 
 /** A trait for REST APIs that write entities to a datastore */
-trait WriteOps extends Directives with Logging {
+trait WriteOps extends Directives {
 
     /** An execution context for futures */
     protected implicit val executionContext: ExecutionContext
 
+    /** logging */
```

Comment:
don't think we need this comment - but if you keep it, how about /** A logging stream. */